### ISSUE TITLE
Invalid spacing applied to custom-selector lists

### ISSUE THREAD

**Issue by jzempel** [csstools/postcss-custom-selectors#40]
Whatever whitespace separator is used between multiple custom-selectors is incorrectly applied to all of the resulting selectors except for the first, which retains the expected spacing.

### Example with no whitespace

```css
@custom-selector :--foo-hover
  .foo:hover;

@custom-selector :--bar-active
  .bar:active;

@custom-selector :--baz-disabled
  .baz.is-disabled;

/* with no space between custom-selectors */
:--foo-hover,:--bar-active,:--baz-disabled {}
```
...produces a valid result:
```css
/* with no space between custom-selectors */
.foo:hover,.bar:active,.baz.is-disabled {}
```

### Example with `SPACE`

```css
@custom-selector :--foo-hover
  .foo:hover;

@custom-selector :--bar-active
  .bar:active;

@custom-selector :--baz-disabled
  .baz.is-disabled;

/* with space between custom-selectors */
:--foo-hover, :--bar-active, :--baz-disabled {}
```
...produces _invalid_ result:
```css
/* with space between custom-selectors */
.foo:hover, .bar :active, .baz .is-disabled {}
```

### Example with `CRLF`

```css
@custom-selector :--foo-hover
  .foo:hover;

@custom-selector :--bar-active
  .bar:active;

@custom-selector :--baz-disabled
  .baz.is-disabled;

/* with newline between custom-selectors */
:--foo-hover,
:--bar-active,
:--baz-disabled {
}
```
...once again, produces _invalid_ result:
```css
/* with newline between custom-selectors */
.foo:hover,
.bar
:active,
.baz
.is-disabled {
}
```

See the following live example for details: https://codepen.io/anon/pen/MZWWzw

It appears this regression appeared between versions [4.0.1 and 5.1.1](https://github.com/postcss/postcss-custom-selectors/compare/cd71c8f150f939c3efb9c15849720c6afcf11648..83292da01b5dab20a7fd5ebd3ab8f18735bfca71) (aka versions [5.3.0 and 6.0.0](https://github.com/csstools/postcss-preset-env/compare/0abb81f6f0bfa70f0f605031b6950d0a383e20de..63f06590a12a2c0eb9663bada9e4eba4efb5d4ab) of `postcss-preset-env`).

**Comment by asifshirazi** [comment]
Hi @jonathantneal! 
It has been quite a while since a release. Is there plan/schedule to fix this issue?

**Comment by romainmenke** [comment]
Hi @jzempel 

This plugin was moved to the csstools mono repo https://github.com/csstools/postcss-plugins/tree/main/plugins/postcss-custom-selectors#readme

A fix for this issue was released as v6.0.1 based on the excellent PR by @georgpukk
